homeassistant/components/light/zha.py

```diff
@@ -105,6 +108,19 @@ def __init__(self, **kwargs):
                 self._supported_features |= light.SUPPORT_COLOR
                 self._hs_color = (0, 0)
 
+    @property
+    def attributes_to_report(self) -> dict:
+        """Return attribute reporting configuration."""
+        return {
```

Comment:
can we replace the magic #'s with constants? or do you feel that makes this too verbose?

Reply:
we could I guess. Would have to add a few different ones and come up with some names, like CONFIG_REPORT_IMMEDIATE, CONFIG_RERPORT_NOT_SO_IMMEDIATE?

Reply:
I was thinking just defining them in each file like: <sensor type>_INITIAL, *_MIN, *_MAX etc... does that make sense?

Reply:
Just pushed new commit. I do like it better now.